main/logkeeper.go

```diff
@@ -34,6 +35,7 @@ func main() {
 	dbHost := flag.String("dbhost", "localhost:27017", "host/port to connect to DB server. Comma separated.")
 	rsName := flag.String("rsName", "", "name of replica set that the DB instances belong to. "+
 		"Leave empty for stand-alone and mongos instances.")
+	localPath := flag.String("localPath", "", "local path to save data to. Omit to save data to S3.")
```

Comment:
We probably will want to make the S3 bucket configurable. I'm guessing we think it would be simpler to make that a second parameter? I think that makes sense...

Reply:
I see, this is configured by the s3options that also gets secrets etc.